config/application.rb

```diff
@@ -24,22 +24,23 @@
 
 module ChaosRails
   class Application < Rails::Application
-    # Settings in config/environments/* take precedence over those specified here.
-    # Application configuration should go into files in config/initializers
-    # -- all .rb files in that directory are automatically loaded after loading
-    # the framework and any gems in your application.
-
     # Initialize configuration defaults for originally generated Rails version.
-    config.load_defaults 7.0
-
-    # Custom directories with classes and modules you want to be autoloadable.
+    config.load_defaults 7.1
+
+    # Please, add to the `ignore` list any other `lib` subdirectories that do
+    # not contain `.rb` files, or that should not be reloaded or eager loaded.
+    # Common ones are `templates`, `generators`, or `middleware`, for example.
+    config.autoload_lib(ignore: %w(assets))
+
+    # Configuration for the application, engines, and railties goes here.
+    #
+    # These settings can be overridden in specific environments using the files
+    # in config/environments, which are processed later.
+    #
+    config.time_zone = "London"
```

Comment:
pretty sure this should be "Europe/London"

Reply:
I had a quick look and London is just an alias for Europe/London. Edinburgh is also a valid alias. https://api.rubyonrails.org/classes/ActiveSupport/TimeZone.html

Reply:
maybe edinburgh then for fun :)